c2corg_api/models/feed.py

```diff
@@ -161,16 +161,12 @@ class DocumentChange(Base):
     more_images = Column(Boolean, server_default='FALSE', nullable=False)
 
     __table_args__ = (
-        # TODO index for enum array? as text?
-        # Index(
-        #     'ix_guidebook_feed_document_changes_activities', activities,
-        #     postgresql_using='gin'),
+        # the queries on the feed table always order by time (desc) and
+        # change_id, therefor create an index for these two columns.
```

Comment:
Just curious if ``therefor`` without the trailing ``e`` is on purpose or not?
I have found this https://www.translegal.com/legal-english-lessons/therefore-vs-therefor